src/views/admin/SchuleCreationView.vue

```diff
@@ -93,14 +102,19 @@
   const onSubmit: (e?: Event | undefined) => Promise<Promise<void> | undefined> = handleSubmit(async () => {
     // TODO: remove this assignment once schulform can be retrieved from the backend.
     // (Necessary here since when we flush the form onSubmit the selectedSchulform won't show up in the success page)
```

Comment:
Der Comment kann jetzt schon raus würde Ich sagen, wenn wir die Orga in Sucess template mit einem find() machen.